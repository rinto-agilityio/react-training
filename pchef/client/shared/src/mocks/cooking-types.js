export const cookingTypes = [
  {
    id: 'baking',
    name: 'Baking',
  },
  {
    id: 'roasting',
    name: 'Roasting',
  },
  {
    id: 'frying',
    name: 'Frying',
  },
]
