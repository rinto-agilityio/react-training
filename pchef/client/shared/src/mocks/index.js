// mock categories
export const category = {
  name: 'Heathy',
  imgUrl:
    'https://images.pexels.com/photos/461198/pexels-photo-461198.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
}
