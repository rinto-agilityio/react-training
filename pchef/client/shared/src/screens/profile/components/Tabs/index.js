import React, { useState } from 'react'

// Components
import { View } from 'react-native'
import TabHeaderItem from '../TabHeaderItem'
import Recipe from '../../../new-feed/components/Recipe'
import Category from '../Category'

// Styles
import styles from './styles'

type Props = {
  categories?: Array<{
    id: string,
    name: string,
    imgUrl: string,
  }>,
  recipes?: Array<{
    id: string,
    title: string,
    imgUrl: string,
    description: string,
    votes: Array<number>,
  }>,
}
const Tabs = ({ categories, recipes }: Props) => {
  // state
  const [tabActive, setTabActive] = useState(0)
  const totalOfRecipes = recipes ? recipes.length : 0
  const totalOfCatefories = categories ? categories.length : 0

  const tabs = [
    { name: 'recipes', number: totalOfRecipes },
    { name: 'favorites', number: totalOfCatefories },
  ]

  const NO_RECIPES_MESSAGE = 'No recipes to show'
  const NO_CATEGORIES_MESSAGE = 'No categories to show'

  return (
    <View>
      <View style={styles.wrapTabs}>
        {tabs.map((tab, index) => (
          <TabHeaderItem
            key={index}
            totalOfItem={tab.number}
            name={tab.name}
            active={tabActive === index}
            handlePressTab={() => setTabActive(index)}
          />
        ))}
      </View>
      <View style={styles.wrapContent}>
        {tabActive === 0 &&
          (!totalOfRecipes
            ? NO_RECIPES_MESSAGE
            : recipes &&
              recipes.map(recipe => (
                <View style={styles.tabContentItem}>
                  <Recipe size="medium" key={recipe.id} recipe={recipe} />
                </View>
              )))}

        {tabActive === 1 &&
          (!totalOfCatefories
            ? NO_CATEGORIES_MESSAGE
            : categories &&
              categories.map(category => (
                <View style={styles.tabContentItem}>
                  <Category key={category.id} category={category} />
                </View>
              )))}
      </View>
    </View>
  )
}

Tabs.defaultProps = {
  recipes: [],
  categories: [],
}

export default Tabs
