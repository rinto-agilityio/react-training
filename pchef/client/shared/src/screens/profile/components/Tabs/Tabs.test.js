// Components
import Tabs from '.'

// Mocks
import { categories, recipes } from '../../../../mocks'

const props = {
  categories,
  recipes,
}

describe('Components', () => {
  describe('Tabs', () => {
    it('should render Tabs component', () => {
      const renderComponent = renderer.create(<Tabs {...props} />)
      expect(renderComponent).toMatchSnapshot()
    })
  })
})
