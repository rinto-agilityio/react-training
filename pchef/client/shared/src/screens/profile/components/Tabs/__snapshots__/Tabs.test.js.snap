// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Tabs should render Tabs component 1`] = `
<div
  className="css-view-1dbjc4n r-marginLeft-1jj8364 r-marginRight-lchren r-width-13qz1uu"
>
  <div
    className="css-view-1dbjc4n r-backgroundColor-14lw9ot r-flexDirection-18u37iz r-marginLeft-1jj8364 r-marginRight-lchren r-maxWidth-c3zab1 r-padding-1pcd2l5 r-paddingBottom-1mdbw0j r-width-13qz1uu"
  >
    <div
      className="css-view-1dbjc4n r-borderBottomWidth-qklmqi r-borderColor-1xc7w19 r-borderStyle-1phboty r-cursor-1loqt21 r-paddingBottom-1inuy60 r-paddingLeft-1hfyk0a r-touchAction-1otgn73 r-width-1iadsmv"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-text-901oao r-fontWeight-majxgm"
        dir="auto"
      >
        4
      </div>
      <div
        className="css-text-901oao r-fontWeight-majxgm"
        dir="auto"
      >
        recipes
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-borderBottomWidth-qklmqi r-borderColor-42olwf r-borderStyle-1phboty r-cursor-1loqt21 r-paddingBottom-1inuy60 r-paddingLeft-1hfyk0a r-touchAction-1otgn73 r-width-1iadsmv"
      data-focusable={true}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      tabIndex="0"
    >
      <div
        className="css-text-901oao r-fontWeight-majxgm"
        dir="auto"
      >
        4
      </div>
      <div
        className="css-text-901oao r-fontWeight-majxgm"
        dir="auto"
      >
        favorites
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n r-backgroundColor-n2h5ot r-flexDirection-18u37iz r-flexWrap-1w6e6rj r-marginLeft-1jj8364 r-marginRight-lchren r-maxWidth-c3zab1 r-minHeight-fo66li r-padding-1pcd2l5 r-width-13qz1uu"
  >
    <div
      className="css-view-1dbjc4n r-paddingLeft-qhyqy2 r-paddingRight-19gegkz r-width-b8tw3c"
    >
      <div
        className="css-view-1dbjc4n r-cursor-1loqt21 r-display-6koalj r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        data-focusable={true}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        tabIndex="0"
      >
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-cqee49 r-fontSize-adyw6z r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            Bun Bo Hue Recipe
          </div>
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-1h2fve2 r-fontSize-1i10wst r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            2lb pork neck bones, 2lb beef bones, 1lb oxtail
          </div>
          <div
            className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-width-13qz1uu r-zIndex-417010"
              style={
                Object {
                  "height": "230px",
                  "marginTop": "10px",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
                style={
                  Object {
                    "backgroundImage": "url(\\"https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg\\")",
                  }
                }
              />
              <img
                alt=""
                className="css-accessibilityImage-9pa8cd"
                draggable={false}
                src="https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-padding-1pcd2l5"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
            <div
              className="css-text-901oao r-marginLeft-1n0xq6e"
              dir="auto"
            >
              Save
            </div>
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
            <div
              className="css-text-901oao r-marginLeft-1n0xq6e"
              dir="auto"
            >
              1
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingLeft-qhyqy2 r-paddingRight-19gegkz r-width-b8tw3c"
    >
      <div
        className="css-view-1dbjc4n r-cursor-1loqt21 r-display-6koalj r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        data-focusable={true}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        tabIndex="0"
      >
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-cqee49 r-fontSize-adyw6z r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            Crispy Fried Chicken
          </div>
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-1h2fve2 r-fontSize-1i10wst r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            Description for Crispy Fried Chicken
          </div>
          <div
            className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-width-13qz1uu r-zIndex-417010"
              style={
                Object {
                  "height": "230px",
                  "marginTop": "10px",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
                style={
                  Object {
                    "backgroundImage": "url(\\"https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg\\")",
                  }
                }
              />
              <img
                alt=""
                className="css-accessibilityImage-9pa8cd"
                draggable={false}
                src="https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-padding-1pcd2l5"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
            <div
              className="css-text-901oao r-marginLeft-1n0xq6e"
              dir="auto"
            >
              Save
            </div>
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingLeft-qhyqy2 r-paddingRight-19gegkz r-width-b8tw3c"
    >
      <div
        className="css-view-1dbjc4n r-cursor-1loqt21 r-display-6koalj r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        data-focusable={true}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        tabIndex="0"
      >
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-cqee49 r-fontSize-adyw6z r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            Crispy Fried Chicken
          </div>
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-1h2fve2 r-fontSize-1i10wst r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            Description for Crispy Fried Chicken
          </div>
          <div
            className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-width-13qz1uu r-zIndex-417010"
              style={
                Object {
                  "height": "230px",
                  "marginTop": "10px",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
                style={
                  Object {
                    "backgroundImage": "url(\\"https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg\\")",
                  }
                }
              />
              <img
                alt=""
                className="css-accessibilityImage-9pa8cd"
                draggable={false}
                src="https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-padding-1pcd2l5"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
            <div
              className="css-text-901oao r-marginLeft-1n0xq6e"
              dir="auto"
            >
              Save
            </div>
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-paddingLeft-qhyqy2 r-paddingRight-19gegkz r-width-b8tw3c"
    >
      <div
        className="css-view-1dbjc4n r-cursor-1loqt21 r-display-6koalj r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
        data-focusable={true}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        tabIndex="0"
      >
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-cqee49 r-fontSize-adyw6z r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            Crispy Fried Chicken
          </div>
          <div
            className="css-text-901oao css-textOneLine-bfa6kz r-color-1h2fve2 r-fontSize-1i10wst r-fontWeight-1kfrs79 r-paddingLeft-b5h31w r-paddingRight-1ah4tor"
            dir="auto"
          >
            Description for Crispy Fried Chicken
          </div>
          <div
            className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-width-13qz1uu r-zIndex-417010"
              style={
                Object {
                  "height": "230px",
                  "marginTop": "10px",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
                style={
                  Object {
                    "backgroundImage": "url(\\"https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg\\")",
                  }
                }
              />
              <img
                alt=""
                className="css-accessibilityImage-9pa8cd"
                draggable={false}
                src="https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1wtj0ep r-padding-1pcd2l5"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
            <div
              className="css-text-901oao r-marginLeft-1n0xq6e"
              dir="auto"
            >
              Save
            </div>
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-901oao r-backgroundColor-1niwhzg r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "WebkitTransform": "scaleX(1)",
                      "color": "rgba(128,128,128,1.00)",
                      "fontFamily": "MaterialIcons",
                      "fontSize": "30px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "transform": "scaleX(1)",
                    }
                  }
                >
                  
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
